code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -31,10 +31,35 @@
         pass_flags = PASSTABLE | PASSMOB
         mob_size = MOB_SIZE_SMALL
         mob_biotypes = list(MOB_ORGANIC, MOB_BEAST, MOB_REPTILE)
+			var/obj/item/udder/snake/udder
         gold_core_spawnable = FRIENDLY_SPAWN
         obj_damage = 0
         environment_smash = ENVIRONMENT_SMASH_NONE
+		
+/mob/living/simple_animal/hostile/retaliate/poison/snake/Initialize()
+	udder = new()
+	. = ..()
 
+/mob/living/simple_animal/hostile/retaliate/poison/snake/Destroy()
+	qdel(udder)
+	udder = null
```

Comment:
```suggestion
```
Why are you setting udder to null after qdel udder?
udder doesn't seem to need a placeholder value.

Reply:
its so it cant be milked after death

Reply:
Is that a bug? You delete the udder right above this line.
What I mean is there is no way for this udder to be used again from what I can tell.